File: app/src/main/java/vergecurrency/vergewallet/service/model/wallet/TxProposalErrorResponse.java
package vergecurrency.vergewallet.service.model.wallet;

public class TxProposalErrorResponse {

	private String code;
	private String message;

	enum Error {
		BAD_SIGNATURES
	}

	public String getCode() {
		return code;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}
}

File: app/src/main/kotlin/vergecurrency/vergewallet/service/model/wallet/TxProposalErrorResponse.kt
package vergecurrency.vergewallet.service.model.wallet

import com.google.gson.Gson

class TxProposalErrorResponse {

    var code: String? = null
    var message: String? = null

    internal enum class Error {
        BAD_SIGNATURES
    }

    companion object {

        fun decode(JSON: String): TxProposalErrorResponse {
            return Gson().fromJson(JSON, TxProposalErrorResponse::class.java)
        }

    }
}
